plutus-pab/src/Plutus/PAB/App.hs

```diff
@@ -123,20 +120,14 @@ runAppBackend trace loggingConfig config action = do
             , nodeClientEnv
             , metadataClientEnv
             , walletClientEnv
-            , signingProcessEnv
             , chainIndexEnv
             } <- mkEnv config
     let
+        Wallet wid = Wallet.wallet $ walletServerConfig config
```

Comment:
I think we talked about this but I forgot the outcome - what is the difference between `newtype Wallet = Wallet Integer`  which we already have, and `type WalletId = Integer`? Could we use `Wallet` for both?